turborepo-tests/integration/tests/command-login.t

```diff
@@ -4,8 +4,3 @@ Setup
 Login Test Run
   $ ${TURBO} login --__test-run
   Login test run successful
-
-Login reuses Vercel CLI token
-  $ . ${TESTDIR}/../../helpers/mock_existing_login.sh
-  $ ${TURBO} login
-  Existing Vercel token found!
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 8 to 12]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This test was good for the previous PRs, but I'm removing it for now until I can figure out a better way to test it without the possibility of having the Github Action hang forever<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/vercel/turbo/7325?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->